OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -63,6 +63,8 @@ public class ArmamentInfo : ConditionalTraitInfo, Requires<AttackBaseInfo>
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		public readonly bool ShootAtClosestTargetablePosition = false;
```

Comment:
I hope the = cell inclusion support would indeed be an option, because I set this up in AS on the day before yesterday  and I really liked what I've seen.